### PR TITLE
Feature/project docs

### PR DESCRIPTION
**Pull request by dannyp303**
- [x] I have reviewed the [OFRAK contributor guide](https://ofrak.com/docs/contributor-guide/getting-started.html) and attest that this pull request is in accordance with it.

**One sentence summary of this PR (This should go in the CHANGELOG!)**

**Link to Related Issue(s)**

**Please describe the changes in your request.**

**Anyone you think should look at this, specifically?**

### CODE REVIEW

File: docs/user-guide/gui/projects.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+![](assets/project-manager.png)
+
+## Creating a Project
+OFRAK projects should be initialized through the OFRAK Project Manager to ensure a correcting project structure. Binaries and scripts are added to the project thorugh the corresponding toolbar options on the left side of the interface. After adding your assets, use the checkboxes in the script panel to link scripts with their binaries. The Save option in the toolbar will save your project, and reset will go back to the previously saved state.
```
Comment (EdwardLarson) [inline docs/user-guide/gui/projects.md:14]:
```suggestion
OFRAK projects should be initialized through the OFRAK Project Manager to ensure a correct project structure. Binaries and scripts are added to the project through the corresponding toolbar options on the left side of the interface. After adding your assets, use the checkboxes in the script panel to link scripts with their binaries. The Save option in the toolbar will save your project, and Reset will go back to the previously saved state.
```

File: docs/user-guide/gui/projects.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Launching a Project Binary
+Binaries from projects can be launched from the Launch option of the toolbar. When launched, the selected script will run on the binary and then display the result in the OFRAK Resource interface.
```
Comment (EdwardLarson) [inline docs/user-guide/gui/projects.md:17]:
```suggestion
Binaries from projects can be launched from the Launch option of the toolbar. When launched, the selected binary's init script (marked by the second checkbox) will run on the binary and then display the result in the OFRAK Resource interface.
```

**Review by EdwardLarson — CHANGES_REQUESTED** [review]

File: docs/user-guide/gui/projects.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+![](assets/project-options.png)
+
+The OFRAK Project Manager interface is loaded upon opening a project and displays the structure of the project along with options to modify it. The Binaries panel displays each binary in the project. Select a binary to work with by clicking on its name. The Script panel shows options for each script in your project. Upon selecting a binary, two checkboxes options appear for each script. Hovering over each checkbox describes its function. One check box selects whether the script is compatible with the currently selected binary, allowing access to it in the Run Script option of the OFRAK Resource toolbar. The second checkbox selects a script to be run upon entering the OFRAK Resource interface, giving a starting point for exploration.
```
Comment (rbs-jacob) [inline docs/user-guide/gui/projects.md:9]:
```suggestion
The OFRAK Project Manager interface is loaded upon opening a project. It displays the structure of the project along with options to modify it. The Binaries panel displays each binary in the project. Select a binary to work with by clicking on its name. The Script panel shows options for each script in your project. Upon selecting a binary, two checkboxes options appear for each script. Hovering over each checkbox describes its function. One check box selects whether the script will run with the currently selected binary, allowing access to it in the Run Script option of the OFRAK Resource toolbar. The second checkbox selects a script to be run upon entering the OFRAK Resource interface, giving a starting point for exploration.
```

**Review by rbs-jacob — CHANGES_REQUESTED** [review]

File: docs/user-guide/gui/projects.md (hunk 3 above)
Comment (dannyp303) [reply docs/user-guide/gui/projects.md]: changed to "can be run on" because i thin "will" makes it sound like it will run on startup

**Review by EdwardLarson — APPROVED** [review]

**Pull request merged**